File: CS/hottrackchart/Form1.cs
using System;
using System.Data;
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace hottrackchart {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Products' table. You can move, or remove it, as needed.
            this.productsTableAdapter.Fill(this.nwindDataSet.Products);
        }

        private void chartControl1_ObjectHotTracked(object sender, DevExpress.XtraCharts.HotTrackEventArgs e) {
            SeriesPoint point = e.AdditionalObject as SeriesPoint;           
            
            if (point != null) {
                DataRowView rowView = (DataRowView)point.Tag;

                string s = "Unit price = " + rowView["UnitPrice"].ToString() +
                    "\r\nUnits in stock = " + rowView["UnitsInStock"].ToString() +
                    "\r\nQuantity per unit = " + rowView["QuantityPerUnit"].ToString();              
                
                toolTipController1.ShowHint(s);
            }
            else
             toolTipController1.HideHint();
        }
    }
}

File: VB/hottrackchart/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Windows.Forms
Imports DevExpress.XtraCharts

Namespace hottrackchart
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'nwindDataSet.Products' table. You can move, or remove it, as needed.
			Me.productsTableAdapter.Fill(Me.nwindDataSet.Products)
		End Sub

		Private Sub chartControl1_ObjectHotTracked(ByVal sender As Object, ByVal e As DevExpress.XtraCharts.HotTrackEventArgs) Handles chartControl1.ObjectHotTracked
			Dim point As SeriesPoint = TryCast(e.AdditionalObject, SeriesPoint)

			If point IsNot Nothing Then
				Dim rowView As DataRowView = CType(point.Tag, DataRowView)

				Dim s As String = "Unit price = " & rowView("UnitPrice").ToString() & Constants.vbCrLf & "Units in stock = " & rowView("UnitsInStock").ToString() & Constants.vbCrLf & "Quantity per unit = " & rowView("QuantityPerUnit").ToString()

				toolTipController1.ShowHint(s)
			Else
			 toolTipController1.HideHint()
			End If
		End Sub
	End Class
End Namespace
